index.html

```diff
@@ -61,7 +65,33 @@ <h2>Mishal Firdous</h2>
         </div>
       </div>
     </div>
-    <script></script>
+    <script>
+      
+      function displayLocalStorage () {
+        const experienceList = document.getElementById("experienceList");
+        console.log(experienceList);
```

Comment:
We should not push console logs to the main branch or even any branch as it slows down the performance, once we are done with debugging, we should remove them.

Reply:
I got it, deleted it from there, and resolved it!